test.js

```diff
@@ -17,12 +18,28 @@ describe('Test Fixtures', () => {
 
   for (let fixture of fixtures) {
     test(fixture, async () => {
+      let writeFileSync;
+      if (fixturesWithFix.includes(fixture)) {
+        process.argv.push('--fix');
+        writeFileSync = fs.writeFileSync;
+        fs.writeFileSync = (filePath, updatedTranslations, encoding) => {
```

Comment:
I'm feeling a bit uneasy about overwriting `fs.writeFileSync()` here. I'm wondering if it would be better to do something similar as we do with `log`, where we pass in the function that is use to writing the files to disk.

Reply:
Updated it to be like log is setup